analyzer/src/funTest/assets/projects/synthetic/dotnet-direct-dependencies-only-expected-output.yml

```diff
@@ -1,6 +1,6 @@
 ---
 project:
-  id: "NuGet::src/funTest/assets/projects/synthetic/dotnet/subProjectTest/test.csproj:"
+  id: "DotNet::src/funTest/assets/projects/synthetic/dotnet/subProjectTest/test.csproj:"
```

Comment:
The following changes in this commit seem to be unrelated to the commit topic.

Reply:
This is no longer relevant with the latest push.